mage/query-modules/cpp/create.md

```diff
@@ -0,0 +1,478 @@
+- `key: string` ➡ name of the property which is about to be set.
+- `value: any` ➡ new value of the property.
+
+
+#### Output:
+
+- `node: Node` ➡ node(s) with modified property.
+
+#### Usage:
+
+```cypher
+CREATE (d:Dog),(h:Human);
+MATCH (d:Dog), (h:Human)
+CALL create.set_property([d,id(h)],"property", 50) YIELD node RETURN node;
+```
+
+```plaintext
++----------------------------+
+| node                       |
++----------------------------+
+| {                          |
+|     "id": 0,               |
+|     "labels": [            |
+|        "Dog"               |
+|     ],                     |
+|     "properties": {        |
+|       property: 50         |
+|     },                     |
+|     "type": "node"         |
+| }                          |
++----------------------------+
+| {                          |
+|     "id": 1,               |
+|     "labels": [            |
+|        "Human"             |
+|     ],                     |
+|     "properties": {        |
+|       property: 50         |
+|     },                     |
+|     "type": "node"         |
+| }                          |
++----------------------------+
+```
+
+### `set_properties(input_nodes, input_keys, input_values)`
+
+Adds the provided properties to the node(s).
+
+#### Input:
+
+- `input_nodes: Any` ➡ node, node's id or a list of nodes and nodes' ids.
+- `input_keys: List[string]` ➡ list of all the property keys to be added to the node(s).
+- `input_values: List[Any]` ➡ list of all the corresponding property values to be added to the node(s).
+
+#### Output:
+
+- `node: Node` ➡ node(s) with new properties.
+
+#### Usage:
+
+```cypher
+CREATE (:Student {name: "Ana"});
+CREATE (:Student {name: "Maria"});
+MATCH (s:Student) CALL create.set_properties(s, ["age", "grade"], [20, "1st"]) YIELD node RETURN node;
+```
+
+```plaintext
++----------------------------+
+| node                       |
++----------------------------+
+| {                          |
+|     "id": 1,               |
+|     "labels": [            |
+|        "Student"           |
+|     ],                     |
+|     "properties": {        |
+|       name: "Ana",         |
+|       age: 20,             |
+|       grade: "1st"         |
+|     },                     |
+|     "type": "node"         |
+| }                          |
++----------------------------+
+| {                          |
+|     "id": 2,               |
+|     "labels": [            |
+|        "Student"           |
+|     ],                     |
+|     "properties": {        |
+|       name: "Maria",       |
+|       age: 20,             |
+|       grade: "1st"         |
+|     },                     |
+|     "type": "node"         |
+| }                          |
++----------------------------+
+```
+
+### `remove_properties(nodes, list_keys)`
+
+Removes all the properties of the given node(s). Input node(s) can be a single node, node ID, or a list of nodes and node IDs. Otherwise, an exception is thrown.
```

Comment:
```suggestion
Removes all the properties of the given node(s). Input node(s) can be a single node, node ID, or a list of nodes and node IDs. Otherwise, an exception is thrown.

```